src/components/common/EmulatorConfigProvider.tsx

```diff
@@ -242,21 +246,21 @@ async function configFetcher(url: string): Promise<Config> {
           // Ditto for IPv6.
           host = '::1';
         }
+        if (key == 'firestore') {
+          // If we remapped the host, apply it to the websocket host too.
+          value.webSocketHost = value.host;
+        }
         break;
       }
     }
+
     result[key as Emulator] = {
       ...value,
       host,
       port,
       hostAndPort: hostAndPort(host, port),
     };
   }
-
-  if (result.firestore?.webSocketPort) {
```

Comment:
previously you only remapped the webSocketPort if it was set, do you need to preserve this if statement?

Reply:
I think it's likely irrelevant (webSocketPort is always provided), but added it back in

Reply:
Did you mean to bring this back?